packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -20,6 +40,20 @@ class FrameFlameChart extends CoreElement {
   CoreElement sectionTitles;
   CoreElement flameChart;
 
+  int _cpuColorOffset = 0;
+  Color get cpuColor {
+    final color = cpuColorPalette[_cpuColorOffset % cpuColorPalette.length];
+    _cpuColorOffset++;
```

Comment:
It looks like this getter is modifying state for the object. You may want to use a regular method, and change the name to something that indicates that multiple calls will change state; `Color nextCpuColor() {...`?

Reply:
done.